cluster/gce/upgrade.sh

```diff
@@ -241,24 +241,31 @@ function prepare-node-upgrade() {
   echo "== Preparing node upgrade (to ${KUBE_VERSION}). ==" >&2
   setup-base-image
 
-  SANITIZED_VERSION=$(echo ${KUBE_VERSION} | sed 's/[\.\+]/-/g')
+  SANITIZED_VERSION="${KUBE_VERSION}//[\.\+]/-/g"
```

Comment:
This seems invalid expression. Perhaps, you mean something like this:
```suggestion
  SANITIZED_VERSION="${KUBE_VERSION//[\.\+]/-}"
```
?